paddle/fluid/operators/lod_reset_op.h

```diff
@@ -35,7 +37,7 @@ class LoDResetKernel : public framework::OpKernel<T> {
       if (lod_t->lod().size() > 0) {
         auto y_lod = lod_t->lod();
         auto last_level = y_lod[y_lod.size() - 1];
-        PADDLE_ENFORCE_EQ(last_level.back(), in->dims()[0],
+        PADDLE_ENFORCE_EQ((int64_t)(last_level.back()), in->dims()[0],
```

Comment:
Would cpplint complain about this C-style cast?

Reply:
No, it doesn't

Reply:
Do you think I should still change it to static_cast

Reply:
It is curious that cpplint doesn't complain. But as it doesn't, I think this PR is great! Thanks!